 Container db  Creating
 db Published ports are discarded when using host network mode 
 Container db  Created
 Container api1  Creating
 Container api2  Creating
 Container api1  Created
 Container api2  Created
 Container marcusantunius-nginx-1  Creating
 Container marcusantunius-nginx-1  Created
Attaching to api1, api2, db, nginx-1
db       | The files belonging to this database system will be owned by user "postgres".
db       | This user must also own the server process.
db       | 
db       | The database cluster will be initialized with locale "en_US.utf8".
db       | The default database encoding has accordingly been set to "UTF8".
db       | The default text search configuration will be set to "english".
db       | 
db       | Data page checksums are disabled.
db       | 
db       | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db       | creating subdirectories ... ok
db       | selecting dynamic shared memory implementation ... posix
db       | selecting default max_connections ... 100
db       | selecting default shared_buffers ... 128MB
db       | selecting default time zone ... Etc/UTC
db       | creating configuration files ... ok
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
db       | running bootstrap script ... ok
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
db       | performing post-bootstrap initialization ... ok
db       | syncing data to disk ... ok
db       | 
db       | 
db       | Success. You can now start the database server using:
db       | 
db       |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db       | 
db       | initdb: warning: enabling "trust" authentication for local connections
db       | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db       | waiting for server to start....2024-02-19 11:30:12.843 UTC [49] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db       | 2024-02-19 11:30:12.843 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db       | 2024-02-19 11:30:12.845 UTC [52] LOG:  database system was shut down at 2024-02-19 11:30:11 UTC
db       | 2024-02-19 11:30:12.848 UTC [49] LOG:  database system is ready to accept connections
db       |  done
db       | server started
db       | CREATE DATABASE
db       | 
db       | 
db       | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/migration.sql
db       | SET
db       | CREATE TABLE
db       | INSERT 0 5
db       | CREATE TABLE
db       | CREATE INDEX
db       | ALTER TABLE
db       | ALTER TABLE
db       | 
db       | 
db       | waiting for server to shut down...2024-02-19 11:30:13.129 UTC [49] LOG:  received fast shutdown request
db       | .2024-02-19 11:30:13.129 UTC [49] LOG:  aborting any active transactions
db       | 2024-02-19 11:30:13.131 UTC [49] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
db       | 2024-02-19 11:30:13.132 UTC [50] LOG:  shutting down
db       | 2024-02-19 11:30:13.132 UTC [50] LOG:  checkpoint starting: shutdown immediate
db       | 2024-02-19 11:30:13.138 UTC [50] LOG:  checkpoint complete: wrote 939 buffers (3.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.006 s, sync=0.001 s, total=0.006 s; sync files=0, longest=0.000 s, average=0.000 s; distance=4289 kB, estimate=4289 kB; lsn=0/191B1D0, redo lsn=0/191B1D0
db       | 2024-02-19 11:30:13.143 UTC [49] LOG:  database system is shut down
db       |  done
db       | server stopped
db       | 
db       | PostgreSQL init process complete; ready for start up.
db       | 
db       | 2024-02-19 11:30:13.247 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db       | 2024-02-19 11:30:13.247 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db       | 2024-02-19 11:30:13.247 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db       | 2024-02-19 11:30:13.247 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db       | 2024-02-19 11:30:13.249 UTC [67] LOG:  database system was shut down at 2024-02-19 11:30:13 UTC
db       | 2024-02-19 11:30:13.252 UTC [1] LOG:  database system is ready to accept connections
nginx-1  | 2024/02/19 11:30:14 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://0.0.0.0:81/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/19 11:30:14 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://0.0.0.0:82/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/19 11:30:16 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/19 11:30:18 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/19 11:30:20 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/19 11:30:22 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/19 11:30:24 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 127.0.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1 exited with code 0
nginx-1 exited with code 0
api2 exited with code 0
api2 exited with code 0
api1 exited with code 0
api1 exited with code 0
db       | 2024-02-19 11:34:40.139 UTC [1] LOG:  received fast shutdown request
db       | 2024-02-19 11:34:40.139 UTC [1] LOG:  aborting any active transactions
db       | 2024-02-19 11:34:40.141 UTC [1] LOG:  background worker "logical replication launcher" (PID 70) exited with exit code 1
db       | 2024-02-19 11:34:40.141 UTC [65] LOG:  shutting down
db       | 2024-02-19 11:34:40.141 UTC [65] LOG:  checkpoint starting: shutdown immediate
db       | 2024-02-19 11:34:40.144 UTC [65] LOG:  checkpoint complete: wrote 685 buffers (2.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.004 s, sync=0.001 s, total=0.004 s; sync files=0, longest=0.000 s, average=0.000 s; distance=2294 kB, estimate=2294 kB; lsn=0/1B58A00, redo lsn=0/1B58A00
db       | 2024-02-19 11:34:40.149 UTC [1] LOG:  database system is shut down
db exited with code 0
